Dockerfile

```diff
@@ -36,12 +36,12 @@ RUN apt-get update \
     powershell
 
 COPY Docker/sshd_config /etc/ssh/sshd_config
-COPY Docker/init.sh init.sh
+COPY Docker/init.sh /home/site/wwwroot/init.sh
 COPY Docker/tokenization.ps1 tokenization.ps1
```

Comment:
add a comment here or provide an alternative that powershell script line will not work in MAC OS